core/src/cg/group4/view/screen/CollectiblesScreen.java

```diff
@@ -0,0 +1,198 @@
+package cg.group4.view.screen;
+
+import java.util.ArrayList;
+
+import cg.group4.rewards.CollectibleDrawer;
+import cg.group4.rewards.Collection;
+import cg.group4.rewards.collectibles.Collectible;
+import cg.group4.rewards.collectibles.collectible_sorters.CollectibleSorter;
+import cg.group4.rewards.collectibles.collectible_sorters.SortByRarity;
+import cg.group4.view.screen_mechanics.ScreenLogic;
+import cg.group4.view.screen_mechanics.ScreenStore;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.ui.CheckBox;
+import com.badlogic.gdx.scenes.scene2d.ui.Image;
+import com.badlogic.gdx.scenes.scene2d.ui.ScrollPane;
+import com.badlogic.gdx.scenes.scene2d.ui.SelectBox;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+import com.badlogic.gdx.scenes.scene2d.ui.WidgetGroup;
+import com.badlogic.gdx.scenes.scene2d.utils.ChangeListener;
+
+/**
+ * Screen to be displayed when pressing the "Collection" button on the home screen.
+ */
+public class CollectiblesScreen extends ScreenLogic {
+	
+	/**
+	 * cContentTable contains the collectibles of the collection.
+	 * cContainer contains the scrollpane displaying the collectibles.
+	 */
+	protected Table cContentTable, cContainer;
+	
+	/**
+	 * ScrollPane displaying the collectibles of the collection.
+	 */
+	protected ScrollPane cScrollPane;
+	
+	/**
+	 * Collection of the player.
+	 */
+	protected Collection cCollection;
+	
+	/**
+	 * Button that takes the player back to the home screen.
+	 */
+	protected TextButton cBackButton;
+	
+	/**
+	 * Numbers of collectibles displayed without scrolling, and the amount of
+	 * checkboxes in the menu.
+	 */
+	protected final int cItemsOnScreen = 10, cNumberOfTopBarItems = 3;
```

Comment:
waar komt die 10 vandaan..? Kun je dat niet af laten hangen van de scherm hoogte/width..? Dank aan een tablet of een old school mini telefoon